src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -117,6 +129,14 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin) {
   /** Flag to enable validation in this component */
   private enableValidation = false
 
+  private get administrativeDissolution (): boolean {
+    return this.name === FilingTypes.ADMINISTRATIVE_DISSOLUTION
+  }
+
+  private get putBackOn (): boolean {
+    return this.name === FilingTypes.PUT_BACK_ON
```

Comment:
Please add both of these to state-getters.ts and import them here instead.

Reply:
'name' is passed as a prop from the parent so I don't think the functions can be moved to state-getters.ts and imported